src/content/language/functions.md

```diff
@@ -620,3 +618,55 @@ is not `final`) an external setter.
 [conditional expression]: /language/operators#conditional-expressions
 [Flutter]: {{site.flutter}}
 [trailing commas]: /language/collections#lists
+
+## Getters and setters
+
+Every property access (top-level, static, or instance) is an invocation of a getter or a setter. A variable implicitly creates a getter and, if it's mutable, a setter. This is why when you access a property, you're actually calling a small function in the background. Reading a property calls a getter function, and writing one calls a setter function, even in cases where the property is declared a variable. 
+
+However, you can also declare a getter or a setter explicitly (using the keywords `get` or `set`). This allows a property's value to be computed when it's read or written.
+
+The purpose of using getters and setters is to create a clear separation between the client (the code that uses the property) and the provider (the class that defines it). The client asks for or sets a value without needing to know if that value is stored in a simple variable or calculated on the spot. This gives the provider the freedom to change how the property works. 
+
+For example, the value of the property might not be stored anywhere, it could be computed each time the getter is called. Another example is when a value is stored in a private variable, and public access is only allowed by calling a getter or a setter. The example below showcases this.
```

Comment:
```suggestion
For example, because the value of the property might not be stored anywhere, it could be computed each time the getter is called. Another example is that when a value is stored in a private variable, public access is only allowed by calling a getter or a setter. The example below showcases this.
```